code/src/constants/routes.ts

```diff
@@ -21,7 +21,23 @@ const routes = {
     path: "/orders"
   },
   dashboard: {
-    path: "/dashboard"
+    path: "/dashboard",
+    orders: {
+      path: "/dashboard/orders",
+      relativePath: "orders"
+    },
+    products: {
+      path: "/dashboard/products",
+      relativePath: "products",
+      new: {
+        path: "/dashboard/products/new",
+        relativePath: "products/new"
+      }
+    },
+    users: {
+      path: "/dashboard/users",
+      relativePath: "users"
+    }
```

Comment:
Excellent job! I am curious about relativePath keys here. I think they can be kinda inferred from path by just joining paths. We can declare only essential part of url and maybe create util to create absolute or relative paths. What do you think about it?

Reply:
Thans for suggestion. I found out, I don't really need relative paths in this case, so I removed all them